docs/application/dotnet/guides/multimedia/raw-audio.md

```diff
@@ -62,18 +64,7 @@ You can enable your application to record uncompressed audio from a microphone-t
 
 Audio data is captured periodically. To receive the audio PCM data from the input device, you must implement the audio capture interface to notify the application of audio data events, such as when the audio data buffer is full.
 
-Before recording audio, you must define the following PCM data settings:
-
--   Input device type:
-    -   Microphone
--   Audio channels:
-    -   Mono (1 channel)
-    -   Stereo (2 channels)
--   Audio sample type:
-    -   Unsigned 8-bit PCM
-    -   Signed 16-bit little endian PCM
--   Audio sample rate:
-    -   8000 \~ 48000 Hz
+Before recording audio, you must define the following PCM data settings. For more information, see [Audio Output](#play_pcm).
```

Comment:
There is no following PCM Data settings below this sentence.

How about removing "following" in the sentence?

Before recording audio, you must define the PCM data settings. For more information, see [Audio Output](#play_pcm).

Reply:
I'll fix it.